docs/contributing/tests.md

```diff
@@ -90,3 +89,87 @@ TEST_DB=cockroachdb TEST_LOCAL=true pnpm --workspace-root test:blackbox
 
 Note: The tests expect the instance running at `localhost:8055`. Make sure to connect the instance to the test database
 container found in the `tests-blackbox/docker-compose.yml` file.
+
+## Writing Unit Tests
```

Comment:
I'd like to have the information about writing tests on a separate page.
See https://github.com/directus/directus/pull/18061/commits/60362294540a1e769c77e8b8d56c95f967fee5a3#diff-811f0eb39dcc489bb4aceb236ea6e0c9e6c9d60cfd862fa16c2121d3bc8e34ed
for how that could look like and also for an explanation of the current structuring of the headers.

Reply:
The goal for this was to pull the up-to-date information about writing tests for what we have into the docs so the GitHub Wiki can be depreciated fully. Then work on creating complete documentation about writing tests. I agree that this needs to be a different page and more in-depth, but I think that needs to be a separate PR/mini-project as I am not as familiar with tests.